Makefile

```diff
@@ -215,6 +239,18 @@ kind-install-underlay-logical-gateway-dual:
 	kubectl taint node kube-ovn-control-plane node-role.kubernetes.io/master:NoSchedule-
 	ENABLE_SSL=true DUAL_STACK=true ENABLE_VLAN=true VLAN_NIC=eth0 LOGICAL_GATEWAY=true ./install-underlay.sh
 
+.PHONY: kind-install-ic
+kind-install-ic:
+	$(eval HOSTIP = $(shell ip -4 route get 8.8.8.8 | awk {'print $$7'} | tr -d '\n'))
+	zone=az0 host_node_ip=${HOSTIP} gateway_node_name=kube-ovn-control-plane j2 yamls/ovn-ic.yaml.j2 -o /ovn-ic-0.yaml
+	zone=az1 host_node_ip=${HOSTIP} gateway_node_name=kube-ovn1-control-plane j2 yamls/ovn-ic.yaml.j2 -o /ovn-ic-1.yaml
+	kubectl config use-context kind-kube-ovn
+	kubectl apply -f /ovn-ic-0.yaml
+	kubectl config use-context kind-kube-ovn1
+	kubectl apply -f /ovn-ic-1.yaml
+	docker run --name=ovn-ic-db -d --network=host -v /etc/ovn/:/etc/ovn -v /var/run/ovn:/var/run/ovn -v /var/log/ovn:/var/log/ovn $(REGISTRY)/kube-ovn:$(RELEASE_TAG) bash start-ic-db.sh
```

Comment:
1. How about moving the commands to `kind-install-cluster` so that we can get an ovn-ic environment using only one command?
2. `ip` command is not available in macos so it won't work in macos. How about creating the ic-db container with network `kind` and pass the container's IP to ovn-ic template?
3. We should remove the ic-db container in `kind-clean`.

Reply:
1, I still prefer to separate the commands based on functionality...
2, u a right. this problem will be solved in ovn-ic db HA test.
3, a kind-clean-cluster is added now.